ui/packages/catalog-ui-search/src/main/webapp/component/announcement/announcement.less

```diff
@@ -89,4 +89,12 @@
     opacity: 0;
     margin: 0px;
   }
+
+  pre {
+      background: inherit;
+      font: inherit;
+      border: none;
+      color: inherit;
```

Comment:
✏️ `font` and `color` are inherited by default, and `border` is none by default

Reply:
Without those explicitly stated, this is what a success message looks like:
![image](https://user-images.githubusercontent.com/39737329/48421566-f9720d00-e719-11e8-8e40-1304cb968ecd.png)